src/types.rs

```diff
@@ -455,6 +579,17 @@ pub struct CompleteObjectInfo {
     /// More information on the object file.
     #[serde(flatten)]
     pub raw: RawObjectInfo,
+
+    /// More information about the DIF files which were consulted for this object file.
+    ///
+    /// For stackwalking and symbolication we need various Debug Information Files about
+    /// this module.  We look for these DIF files in various locations, this describes all
+    /// the DIF files we looked up and what we know about them, how we used them.  It can be
+    /// helpful to understand what information was available or missing and for which
+    /// reasons.
+    ///
+    /// This list is not serialised if it is empty.
+    pub candidates: Vec<ObjectCandidate>,
```

Comment:
I tried adding `#[serde(skip_serializing_if = "Vec::is_empty")]` here but that failed in very strange ways where it would never terminate, I stopped trying to debug it but it might still be worth to re-visit this.